src/tests/ut/patient/patient_test.cpp

```diff
@@ -0,0 +1 @@
+#include "patient_test.hpp"
```

Comment:
I think your suggestion was to put actual tests here.

As the test fixture is small IMO we have 2 options:
1. put everything in .cpp file
2. split fixture into .hpp file and actual tests into .cpp file.

Reply:
yep
I follow that approach, however at the point when you placed your comments PR wasnt' ready, but I couldn't switch back to draft state